### DISCUSSION TITLE
Permission 'storage.objects.get' denied on resource (or it may not exist).

### DISCUSSION THREAD

**Discussion by mdeprezzo** [spatie/laravel-google-cloud-storage#65] [Q&A]
Hi all guys,
i've a question, when i generate standard url like this: $disk->url('folder/file.pdf') and i try to download the file, i get this message:

Anonymous caller does not have storage.objects.get access to the Google Cloud Storage object. Permission 'storage.objects.get' denied on resource (or it may not exist).

what am i missing?

**Comment by yudha47** [comment]
> Hi all guys, i've a question, when i generate standard url like this: $disk->url('folder/file.pdf') and i try to download the file, i get this message:
> 
> Anonymous caller does not have storage.objects.get access to the Google Cloud Storage object. Permission 'storage.objects.get' denied on resource (or it may not exist).
> 
> what am i missing?

i got the same error, do you find the solution ?
